save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -622,22 +630,22 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                     div {
                         className = ClassName("ml-3 mt-2 align-items-left justify-content-between")
                         fontAwesomeIcon(icon = faHistory)
-
-                        button {
-                            className = ClassName("btn btn-link text-left")
-                            +"Latest Execution"
-                            disabled = state.latestExecutionId == null
-
-                            onClick = {
-                                window.location.href = "${window.location}/history/execution/${state.latestExecutionId}"
+                        withNavigate { navigateContext ->
+                            button {
+                                className = ClassName("btn btn-link text-left")
+                                +"Latest Execution"
+                                disabled = state.latestExecutionId == null
+                                onClick = {
+                                    navigateContext.generateLinksWithSuffix<ProjectMenuBar>(state.paths.pathDefaultTab, "history/execution/${state.latestExecutionId}")
+                                }
                             }
                         }
                     }
                     div {
                         className = ClassName("ml-3 align-items-left")
                         fontAwesomeIcon(icon = faCalendarAlt)
                         a {
-                            href = "#/${state.project.organization.name}/${state.project.name}/history"
```

Comment:
Why slash in the beginning of the value is removed?

Reply:
Sorry, added this

Reply:
```pathDefaultTab``` start with ```\```

Need to store it ```pathDefaultTab```, for a good transition to other pages, using ```navigate``` (the url should not start with ```#```, but with ```/```). Therefore, it is used everywhere, both for navigating by ```navigate```, and for the usual change of href when switching between tabs(change ```window.location.href```)